scripts/ccpp_prebuild_config_SCM.py

```diff
@@ -17,52 +17,52 @@
-    'ccpp-physics/physics/radiation_gases.f',
-    'ccpp-physics/physics/radiation_surface.f',
-    'ccpp-physics/physics/radlw_datatb.f',
-    'ccpp-physics/physics/radlw_param.f',
-    'ccpp-physics/physics/radsw_datatb.f',
-    'ccpp-physics/physics/radsw_param.f',
-    'ccpp-physics/physics/rascnvv2.f',
-    'ccpp-physics/physics/set_soilveg.f',
-    'ccpp-physics/physics/sflx.f',
-    'ccpp-physics/physics/surface_perturbation.F90',
-    'ccpp-physics/physics/cu_gf_deep.F90',
-    'ccpp-physics/physics/cu_gf_sh.F90',
-    'ccpp-physics/physics/tridi.f',
-    'ccpp-physics/physics/wam_f107_kp_mod.f90',
-    'ccpp-physics/physics/module_sf_ruclsm.F90',
-    'ccpp-physics/physics/namelist_soilveg_ruc.F90',
-    'ccpp-physics/physics/set_soilveg_ruc.F90',
-    'ccpp-physics/physics/module_soil_pre.F90',
-    'ccpp-physics/physics/wv_saturation.F',
+    'ccpp/physics/physics/GFDL_parse_tracers.F90',
+    'ccpp/physics/physics/aer_cloud.F',
+    'ccpp/physics/physics/calpreciptype.f90',
+    'ccpp/physics/physics/cldwat2m_micro.F',
+    'ccpp/physics/physics/date_def.f',
+    'ccpp/physics/physics/funcphys.f90',
+    'ccpp/physics/physics/gfs_phy_tracer_config.f',
+    'ccpp/physics/physics/gocart_tracer_config_stub.f',
+    'ccpp/physics/physics/h2o_def.f',
+    'ccpp/physics/physics/h2ointerp.f90',
+    'ccpp/physics/physics/iounitdef.f',
+    'ccpp/physics/physics/machine.F',
+    'ccpp/physics/physics/mersenne_twister.f',
+    'ccpp/physics/physics/mfpbl.f',
+    'ccpp/physics/physics/module_bfmicrophysics.f',
+    'ccpp/physics/physics/module_nst_model.f90',
+    'ccpp/physics/physics/module_nst_parameters.f90',
+    'ccpp/physics/physics/module_nst_water_prop.f90',
+    'ccpp/physics/physics/namelist_soilveg.f',
+    'ccpp/physics/physics/ozinterp.f90',
+    'ccpp/physics/physics/ozne_def.f',
+    'ccpp/physics/physics/physcons.F90',
+    'ccpp/physics/physics/physparam.f',
+    'ccpp/physics/physics/radcons.f90',
+    'ccpp/physics/physics/radiation_aerosols.f',
+    'ccpp/physics/physics/radiation_astronomy.f',
+    'ccpp/physics/physics/radiation_clouds.f',
+    'ccpp/physics/physics/radiation_gases.f',
+    'ccpp/physics/physics/radiation_surface.f',
+    'ccpp/physics/physics/radlw_datatb.f',
+    'ccpp/physics/physics/radlw_param.f',
+    'ccpp/physics/physics/radsw_datatb.f',
+    'ccpp/physics/physics/radsw_param.f',
+    'ccpp/physics/physics/rascnvv2.f',
+    'ccpp/physics/physics/set_soilveg.f',
+    'ccpp/physics/physics/sflx.f',
+    'ccpp/physics/physics/surface_perturbation.F90',
+    'ccpp/physics/physics/cu_gf_deep.F90',
+    'ccpp/physics/physics/cu_gf_sh.F90',
+    'ccpp/physics/physics/tridi.f',
+    'ccpp/physics/physics/wam_f107_kp_mod.f90',
+    'ccpp/physics/physics/module_sf_ruclsm.F90',
+    'ccpp/physics/physics/namelist_soilveg_ruc.F90',
+    'ccpp/physics/physics/set_soilveg_ruc.F90',
+    'ccpp/physics/physics/module_soil_pre.F90',
+    'ccpp/physics/physics/wv_saturation.F',
     'scm/src/GFS_typedefs.F90',
```

Comment:
Why is this file still in the repo? I thought the whole idea was to separate host model information from the framework.

Reply:
Dito. This PR moves the SDFs out, nothing else.